### DISCUSSION TITLE
LCARS Watchface

### DISCUSSION THREAD

**Discussion by infinitytec** [InfiniTimeOrg/InfiniTime#1661] [Ideas]
I think it would be neat to have an LCARS (computer interfaces from various Star Trek shows and films) themed watchface. This has been [done before](https://www.youtube.com/watch?v=1AmsDaFMEF0) with Wasp OS as can be seen on the PineTime page on the Pine64 website.

I'd be willing to help, but do not have much experience with C++.

**Comment by FintasticMan** [comment]
Someone has done this a couple of months ago, but it hasn't been updated much recently: #1462.

**Reply by infinitytec** [reply]
Wow! That's fantastic!

**Reply by L3br4nd** [reply]
See also #2072
